common/src/jni/main/cpp/conscrypt/native_crypto.cc

```diff
@@ -9861,6 +9861,34 @@ static jstring NativeCrypto_SSL_get_current_cipher(JNIEnv* env, jclass, jlong ss
     return env->NewStringUTF(name);
 }
 
+static void NativeCrypto_SSL_set1_curve_list(JNIEnv* env, jclass, jlong sslAddress,
+                                             CONSCRYPT_UNUSED jobject sslHolder,
+                                             jstring curveNameList) {
+    CHECK_ERROR_QUEUE_ON_RETURN;
+    SSL* ssl = to_SSL(env, sslAddress, true);
+    JNI_TRACE("ssl=%p NativeCrypto_SSL_set1_curve_list curves=%p", ssl, curveNameList);
+    if (ssl == nullptr) {
+        return;
+    }
+    if (curveNameList == nullptr) {
+        conscrypt::jniutil::throwNullPointerException(env, "curveNameList == null");
+        return;
+    }
+
+    ScopedLocalRef<jstring> curve(env, curveNameList);
+    ScopedUtfChars c(env, curve.get());
+    if (c.c_str() == nullptr) {
+        conscrypt::jniutil::throwNullPointerException(env, "c.c_str() == null");
+        return;
+    }
+
+    if (!SSL_set1_curves_list(ssl, c.c_str())) {
```

Comment:
It seems BoringSSL wants you to call this differently:

http://google3/third_party/openssl/boringssl/src/include/openssl/ssl.h;l=5920-5922;rcl=829481133

http://google3/third_party/openssl/boringssl/src/include/openssl/ssl.h;l=5964;rcl=829481133

So SSL_set1_groups_list ?